corgidrp/l1_to_l2a.py

```diff
@@ -205,9 +206,12 @@ def detect_cosmic_rays(input_dataset, detector_params, sat_thresh=0.7,
 
     crmasked_cube = crmasked_dataset.all_data
 
-
     # Calculate the full well capacity for every frame in the dataset
-    kgain = np.array([detector_params.params['kgain'] for frame in crmasked_dataset])
+    if k_gain == None:
```

Comment:
should use `if k_gain is None` (forget the exact reason, but `is` is preferred to `==`)